tests/linker/ios/link sdk/CryptoTest.cs

```diff
@@ -152,42 +153,37 @@ public void TLS1_ServerNameExtension ()
 			
 			X509Certificate2 cert = new X509Certificate2 (certData);
 			X509Chain chain = new X509Chain (false);
+
+#if NET
+			chain.ChainPolicy.TrustMode = X509ChainTrustMode.CustomRootTrust;
+			chain.ChainPolicy.CustomTrustStore.Add (rootCert);
+#endif
+
 			Assert.False (chain.Build (cert), "Online");
+			Assert.True (chain.ChainStatus.Any (s => s.Status.HasFlag (X509ChainStatusFlags.RevocationStatusUnknown)), "Online");
 
 			chain.ChainPolicy.RevocationMode = X509RevocationMode.Offline;
 			Assert.False (chain.Build (cert), "Offline");
+			Assert.True (chain.ChainStatus.Any (s => s.Status.HasFlag (X509ChainStatusFlags.RevocationStatusUnknown)), "Offline");
 
 			chain.ChainPolicy.RevocationMode = X509RevocationMode.NoCheck;
-#if __MACOS__
-			// Not sure if this is expected on macOS, or if it's a bug somewhere in our code.
-			Assert.False (chain.Build (cert), "NoCheck");
-#else
 			Assert.True (chain.Build (cert), "NoCheck");
-#endif
+			Assert.True (chain.ChainStatus.Length == 0, "NoCheck");
```

Comment:
Minor, but this answers the inevitable question "but what was the length?" if this ever fails, since NUnit will log the actual value:

```suggestion
			Assert.AreEqual (0, chain.ChainStatus.Length, "NoCheck");
```